ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -322,17 +322,27 @@ StatusSectionLayout {
                    value: communityMintTokensSettingsPanel.StackView.index
                 }
 
-                // TODO: Review once backend is done
                 Connections {
                     target: rootStore.communityTokensStore
-                    function onDeployFeeUpdated(value) {
-                        // TODO better error handling
-                        if (value === "-") {
-                            mintPanel.isFeeLoading = true
-                        } else {
+                    function onDeployFeeUpdated(ethFee, fiatFee, errorCode) {
+                        if (errorCode === Constants.ComputeFeeErrorCode.Success || errorCode === Constants.ComputeFeeErrorCode.Balance) {
+                            let ethRounded = (ethFee).toFixed(4)
+                            let fiatRounded = (fiatFee).toFixed(2)
+                            let valueStr = ethRounded + "ETH ("+ fiatRounded + rootStore.currentCurrency +")"
```

Comment:
This deserves some proper formatting :)

Reply:
What do you mean ?

Reply:
or spaces between math operations? :)

Reply:
Ok, I fixed this and use LocaleUtils.currencyAmountToLocaleString now.

Reply:
Great, thanks!